assets/javascripts/redmine_s3.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This script handles thumbnail image missing errors
+ */
+
+$(function() {
```

Comment:
The thumbnails generation should be done via rails console in this case, rake task or some other tool.